views/macros/ul-checkmark.njk

```diff
@@ -0,0 +1,21 @@
+{#
+    Shows a list with items prefixed by a checkmark (for true) or "X" (for false) along with some text.
+
+    Parameters:
+        - items: 2 element arrays, with the first element being the translation key for text and the second being true (checkmark) or false ("X")
+#}
+
+{% macro checkmarkList (items) %}
+    <ul class="list-none">
+        {% for key, isCheckmark in items %}
+            <li>
+                {% if isCheckmark %}
+                    <svg class="inline h-5 w-5" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M8.294 16.998c-.435 0-.847-.203-1.111-.553L3.61 11.724a1.392 1.392 0 0 1 .27-1.951 1.392 1.392 0 0 1 1.953.27l2.351 3.104 5.911-9.492a1.396 1.396 0 0 1 1.921-.445c.653.406.854 1.266.446 1.92L9.478 16.34a1.39 1.39 0 0 1-1.12.656c-.022.002-.042.002-.064.002z"/></svg>
+                {% else %}
+                    <svg class="inline h-5 w-5" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"/></svg>
+                {% endif %}
+                &nbsp;{{ __(key) }}
```

Comment:
instead of a nbsp, I think you could add a `whitespace-no-wrap` class to the parent element?

Reply:
Will do!

Reply:
Just removed the nbsp and it looks fine (even without the class). Good catch!

Reply:
I have a (probably undeserved) aversion to nbsp.